detox/src/android/espressoapi/DetoxAction.js

```diff
@@ -233,6 +233,18 @@ class DetoxAction {
     };
   }
 
+  static parseDateISO8601(dateString) {
```

Comment:
@d4vidi looks like we need to remove `private` methods generation, right?

Reply:
No, we have to switch to `invoke2` 😄